### DISCUSSION TITLE
[Feature Request] Linear Interpolation instead of step plot

### DISCUSSION THREAD

**Discussion by schoppfe** [home-assistant/frontend#23385] [Dashboard]
Dear Home Assistant Team,

Thank you for your fantastic work on this platform!

Currently, graphs in Home Assistant default to step plots, which can be less intuitive for continuous data, such as temperatures. Please see the attached screenshot as an example: the blue step plot contrasts with the red line, which uses linear interpolation for smoother trend visualization.

I would like to suggest adding an option to enable linear interpolation for graphs. This would greatly improve the readability of continuous data and provide users with more flexibility in data representation.

Thank you for considering this enhancement!

<img width="540" alt="image" src="https://github.com/user-attachments/assets/a3017505-a4cf-4970-a17d-8365b396a379" />

**Comment by ildar170975** [comment]
Note that these interpolated data are fake.
